### ISSUE TITLE
Feature request: Gravis Ultrasound sound card emulation

### ISSUE THREAD

**Issue by maximilien-noal** [OpenRakis/Spice86#319]
If there is real mode games that use it ? (I don't know. I think so).

**Comment by xcom169** [comment]
How can we know about a game / Exe that is it a 286, 16 bit, real mode, or  something upper?

**Comment by maximilien-noal** [comment]
One cannot know, DOS apps don't identify themselves, and can mix any memory addressing mode or CPU features.

If they have DOS4GW or other well known DPMI server shipped with them, they probably use protected mode, but that's about it.

You'll have to study them. Or run them with PCEM/86Box, which can emulates exact CPU and memory hardware (such as a 286, a 386, a 486, a Pentium, ...)

**Comment by xcom169** [comment]
Okay. But Spice86 can only run real mode games like Stunts, Dune 1 or Prehistorik 1 maybe?

**Comment by maximilien-noal** [comment]
Indeed, Spice86 only runs real mode apps (either PC booter, DOS app, or BIOS).

**Comment by maximilien-noal** [comment]
https://github.com/dosbox-staging/dosbox-staging for gravis ultrasound reference implementation.
